Description: Don't use libiberty
Author: Bastian Blank <waldi@debian.org>
Bug-Debian: http://bugs.debian.org/730910
---
--- ipxe-1.0.0+git-20131111.c3d1e78.orig/src/Makefile.housekeeping
+++ ipxe-1.0.0+git-20131111.c3d1e78/src/Makefile.housekeeping
@@ -1179,7 +1179,7 @@ CLEANUP += $(ZBIN)
 ELF2EFI_CFLAGS	:= -I$(BINUTILS_DIR)/include -I$(BFD_DIR)/include \
 		   -I$(ZLIB_DIR)/include -idirafter include
 ELF2EFI_LDFLAGS	:= -L$(BINUTILS_DIR)/lib -L$(BFD_DIR)/lib -L$(ZLIB_DIR)/lib \
-		   -lbfd -ldl -liberty -lz -Wl,--no-warn-search-mismatch
+		   -lbfd -ldl -lz -Wl,--no-warn-search-mismatch
 
 $(ELF2EFI32) : util/elf2efi.c $(MAKEDEPS)
 	$(QM)$(ECHO) "  [HOSTCC] $@"
